spec/services/meal_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Meal do
+  let(:params) do
+    { params: { name: 'pizza',
+                course: 'main course',
+                price: 42,
+                id: 1 },
+      menu: Menu.create(id: 1) }
```

Comment:
Why did you forcefully set id to menu?
Also read about `FactoryGirl` and `Faker`
[Factory Girl](https://github.com/thoughtbot/factory_bot)
[Faker](https://github.com/stympy/faker)